netdata-installer.sh

```diff
@@ -1320,11 +1320,30 @@ should_install_ebpf() {
   return 0
 }
 
+remove_old_ebpf() {
+  if [ -f "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/ebpf_process.plugin" ]; then
+    echo "Removing alpha eBPF collector."
+    rm -rf "${NETDATA_PREFIX}/usr/libexec/netdata/plugins.d/ebpf_process.plugin" 
+  fi
+
+  if [ -f "${NETDATA_PREFIX}/usr/lib/netdata/conf.d/ebpf_process.conf" ]; then
+    echo "Removing alpha eBPF stock file"
+    rm -rf "${NETDATA_PREFIX}/usr/lib/netdata/conf.d/ebpf_process.conf" 
+  fi
+
+  if [ -f "${NETDATA_PREFIX}/etc/netdata/ebpf_process.conf" ]; then
+    echo "Renaming eBPF configuration file."
```

Comment:
You should probably redirect to stderr like we do in most places (_although not consistetntly it seems_):

```#!sh
$ sift -N --only-matching 'echo (>&2|)' netdata-installer.sh | sort | uniq -c
     23 echo
     53 echo >&2
```

Reply:
I changed it now, thank you!